### PR TITLE
Fix: Issues identified using helm-template

### PR DESCRIPTION
**Pull request by oseasmoran73**
There were some issues identified the first time around. This PR fixes them

https://github.com/department-of-veterans-affairs/va.gov-team/issues/46184

### CODE REVIEW

File: .github/workflows/application-onboarding-helm-template.yml [hunk 1]
```diff
@@ -81,9 +88,23 @@ jobs:
...
+
       - name: Modify ingress environment to use dev
         if: steps.directory_check.outputs.directory_exists == 'false'
-        run: cat <<< $(yq -e '(.ingress.environment)="${{ inputs.application_environment }}"' vsp-infra-application-manifests/apps/om-test/dev/values.yaml) > vsp-infra-application-manifests/apps/om-test/dev/values.yaml
+        run: cat <<< $(yq -e '(.ingress.environment)="${{ inputs.application_environment }}"' vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/values.yaml) > vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/values.yaml
+
+      - name: Enable ingress
+        if: steps.directory_check.outputs.directory_exists == 'false'
+        run: cat <<< $(yq -e '(.ingress.enabled)=true' vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/values.yaml) > vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/values.yaml
```
Comment (jbritt1) [inline .github/workflows/application-onboarding-helm-template.yml:101]: Why not just make it enabled: true in the template?

Comment (oseasmoran73) [reply .github/workflows/application-onboarding-helm-template.yml:101]: 🤔 Idk lol. Changing it now

Comment (oseasmoran73) [reply .github/workflows/application-onboarding-helm-template.yml]: Done

File: .github/workflows/application-onboarding-helm-template.yml [hunk 2]
```diff
@@ -73,6 +76,10 @@ jobs:
         if: steps.directory_check.outputs.directory_exists == 'false'
         run: cp vsp-infra-helm/hello-world/Chart.yaml vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/Chart.yaml
 
+      - name: Modify Chart.yaml to have correct format
```
Comment (rmtolmach) [inline .github/workflows/application-onboarding-helm-template.yml:79]: What's the correct format?

Comment (oseasmoran73) [reply .github/workflows/application-onboarding-helm-template.yml:79]: Here is an [example](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/main/apps/datadog-agent/dev/values.yaml) of correct format. Needs app-name then information

**Comment by oseasmoran73** [comment]
@jbritt1 @rmtolmach Feedback has been addressed. Can I get a second passing?

**Review by rmtolmach — APPROVED** [review]

**Pull request merged**